src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -83,9 +83,14 @@ private string CreateBugFooter()
             if(this.Context.CurrentProvider == FilingClient.SourceControlProvider.AzureDevOps)
             {
                 StringBuilder azureDevOpsFooter = new StringBuilder();
-                azureDevOpsFooter.Append(@"To see result details, please visit the Scans tab of this bug, or the attached SARIF log.");
+                azureDevOpsFooter.Append(@"<br><br>Other viewing options:<br>");
```

Comment:
><br> [](start = 43, length = 4)

I take it ADO's work item text fields accept HTML rather than Markdown. I seem to remember us talking about that a few days ago.

Reply:
yeah, ADO is HTML, github is markdown.  I think the long term solution is to do all markdown and then pass it through a markdown->html converter, which I've seen a couple of.  We'd just want to evaluate which one we would use.

---
In reply to: [398186438](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r398186438) [](ancestors = 398186438)